docs/CONFIG.md

```diff
@@ -4636,7 +4636,7 @@ SelectionMode controls node selection strategy:
 SyncThreshold = 5 # Default
 ```
 SyncThreshold controls how far a node may lag behind the best node before being marked out-of-sync.
-Depending on `SelectionMode`, this represents a difference in the number of blocks (`HighestHead`, `RoundRobin`), or total difficulty (`TotalDifficulty`).
+Depending on `SelectionMode`, this represents a difference in the number of blocks (`HighestHead`, `RoundRobin`,`PriorityLevel`), or total difficulty (`TotalDifficulty`).
```

Comment:
```suggestion
Depending on `SelectionMode`, this represents a difference in the number of blocks (`HighestHead`, `RoundRobin`, `PriorityLevel`), or total difficulty (`TotalDifficulty`).
```

Reply:
Fixed in [`2c3b889` (#9329)](https://github.com/smartcontractkit/chainlink/pull/9329/commits/2c3b889363a9a0274b0163340157234ab4b86162)